cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/SlangSource.java

```diff
@@ -38,19 +40,33 @@ public static SlangSource fromFile(File file) {
 
         String source;
         try {
-            source = FileUtils.readFileToString(file, Charset.defaultCharset());
+            source = readFileToString(file);
         } catch (IOException e) {
             throw new RuntimeException("There was a problem reading the yaml file: " + file.getName(), e);
         }
         return new SlangSource(source, file.getName());
     }
 
+    public static String readFileToString(File file) throws IOException {
```

Comment:
given the factbthat for now only files that end with `.sl` are considered slang sources (inputs files are not) + this is a general utility method, maybe you should create a new utility class for this method